cachito/web/models.py

```diff
@@ -242,6 +271,22 @@ def from_json(cls, kwargs):
 
             request_kwargs['flags'] = found_flags
 
+        dependency_overrides = request_kwargs.get('dependency_overrides', [])
+        if not isinstance(dependency_overrides, list):
+            raise ValidationError('"dependency_overrides" must be an array')
+
+        dependency_override_objects = []
+        for dep in dependency_overrides:
```

Comment:
I prefer having the entire name `dependency` instead of `dep`.

I blame luiz's book for that. :p

Reply:
But I'll leave it up to you :)

Reply:
Sure, I'll use `dependency_override` to be more descriptive.